enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -593,7 +594,7 @@ def _prepare_response_socket(self):
         self.log.debug("Response socket launched on {}, port: {} using {}s timeout".format(local_ip, port, socket_timeout))
```

Comment:
The debug statement should reflect the proper address.  Probably best to move the debug statement and reformat (i.e., remove the `, port:{}` entry).